src/components/SongEntry/SongEntry.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { render, screen } from '@testing-library/react'
+import SongEntry from './SongEntry'
+
+const artistsArray: SpotifyApi.ArtistObjectSimplified[] = [
+  {
+    external_urls: { spotify: 'test.external.de' },
+    href: 'test.href.de',
+    id: '1',
+    name: 'John Doe',
+    type: 'artist',
+    uri: 'test.uri.de',
+  },
+  {
+    external_urls: { spotify: 'testTwo.external.de' },
+    href: 'testTwo.href.de',
+    id: '2',
+    name: 'Jane Doe',
+    type: 'artist',
+    uri: 'testTwo.uri.de',
```

Comment:
You either don't need all this properties or you did not test them properly. You only check for the names below.